src/simd_128.h

```diff
@@ -40,17 +40,17 @@ typedef __m128i VecType;
 
 /* ============= Constant variable  ============ */
 
-#define F4_u32 _mm_set1_epi32(65537)
-#define F4m1_u32 _mm_set1_epi32(65536)
-#define F3_u32 _mm_set1_epi32(257)
-#define F3m1_u32 _mm_set1_epi32(256)
+#define F4_U32 _mm_set1_epi32(65537)
```

Comment:
Good, but we shouldn't have a patch for renaming: you should fix the naming directly in the patch that introduce them (with an interactive rebase)